deployment/modules/cloudbuild/main.tf

```diff
@@ -0,0 +1,97 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "5.14.0"
```

Comment:
Lastest version is 5.41. Would recommend using the latest version (if not now, in a follow-up PR). Usually there are no changes needed, but with this many releases, there could be...

https://registry.terraform.io/providers/hashicorp/google/latest

Relatedly, but orthogonally, we should look at dependabot for these config bumps. https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#terraform suggests the feature is there, but the TL;DR of https://github.com/dependabot/dependabot-core/issues/8725 is "use renovate".